flake.nix

```diff
@@ -0,0 +1,80 @@
+{
+  description = "BizHawk is a multi-system emulator written in C#. BizHawk provides nice features for casual gamers such as full screen, and joypad support in addition to full rerecording and debugging tools for all system cores.";
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/release-24.05";
+  };
+  outputs =
+    inputs@{ self, nixpkgs, ... }:
+    with builtins;
+    let
+      std = nixpkgs.lib;
```

Comment:
Can't you just add `lib` to the function paramset of `outputs`?

Reply:
I don't think so? As far as I can tell, the only inputs to `outputs` are `self` and whatever inputs you define in the `inputs` attrset.

Reply:
On mine that gives me `error: cannot find flake 'flake:lib' in the flake registries`; I think that's adding an implicit flake input like:
 ```nix
 inputs.lib = {
    type = "indirect";
    id = "lib";
};
```
and, for some reason, you have `lib` in your registry and I don't.